BizHawk.Client.EmuHawk/tools/Lua/Libraries/EmuLuaLibrary.Tastudio.cs

```diff
@@ -23,6 +23,32 @@ public TastudioLuaLibrary(Lua lua, Action<string> logOutputCallback)
 
 		private TAStudio Tastudio => GlobalWin.Tools.Get<TAStudio>() as TAStudio;
 
+		private struct PENDING_CHANGES
+		{
+			public LUA_CHANGE_TYPES type;
+			public INPUT_CHANGE_TYPES inputType;
+			public int frame;
+			public int number;
+			public string button;
+			public bool valueBool;
+			public float valueFloat;
+		};
+
+		public enum LUA_CHANGE_TYPES
+		{
+			INPUTCHANGE,
+			INSERTFRAMES,
+			DELETEFRAMES,
+		};
+
+		public enum INPUT_CHANGE_TYPES
+		{
+			BOOL,
+			FLOAT,
+		};

```

Comment:
Use CamelCase for enums (names and members) instead of caps. Otherwise, should be fine: this is unlikely to break anything, and once issues are found, we'll fix them.